File: SemantikosWSEJB/src/cl/minsal/semantikos/ws/response/CrossmapSetMembersResponse.java
package cl.minsal.semantikos.ws.response;

import cl.minsal.semantikos.model.crossmaps.Crossmap;
import cl.minsal.semantikos.model.crossmaps.CrossmapSetMember;
import cl.minsal.semantikos.model.crossmaps.IndirectCrossmap;

import javax.xml.bind.annotation.*;
import java.util.ArrayList;
import java.util.List;

/**
 * @author Andrés Farías on 12/13/16.
 */
@XmlAccessorType(XmlAccessType.FIELD)
@XmlRootElement(name = "crossmapSetMembersResponse", namespace = "http://service.ws.semantikos.minsal.cl/")
@XmlType(name = "CrossmapSetMembersResponse", namespace = "http://service.ws.semantikos.minsal.cl/")
public class CrossmapSetMembersResponse {

    /** La lista de crossmaps indirectos (response) */
    @XmlElementWrapper(name = "crossmapSetMembers")
    @XmlElement(name = "crossmapSetMember")
    private List<CrossmapSetMemberResponse> crossmapSetMemberResponses;

    public CrossmapSetMembersResponse() {
        this.crossmapSetMemberResponses = new ArrayList<>();
    }

    /**
     * Este constructor es responsable de poblar la lista de crossmapsSetMembers "response" a partir del  objeto de
     * negocio.
     *
     * @param crossmapSetMembers La lista de crossmapSetMembers de negocio.
     */
    public CrossmapSetMembersResponse(List<CrossmapSetMember> crossmapSetMembers) {
        this();

        for (CrossmapSetMember crossmapSetMember : crossmapSetMembers) {
            this.crossmapSetMemberResponses.add(new CrossmapSetMemberResponse(crossmapSetMember));
        }
    }

    public List<CrossmapSetMemberResponse> getCrossmapSetMemberResponses() {
        return crossmapSetMemberResponses;
    }

    public void setCrossmapSetMemberResponses(List<CrossmapSetMemberResponse> crossmapSetMemberResponses) {
        this.crossmapSetMemberResponses = crossmapSetMemberResponses;
    }
}

File: SemantikosClientesWS/clientePHP/src/Semantikos/ClientBundle/API/CrossmapSetMembersResponse.php
<?php

namespace Semantikos\ClientBundle\API;

class CrossmapSetMembersResponse
{

    /**
     * @var crossmapSetMembers $crossmapSetMembers
     */
    protected $crossmapSetMembers = null;

    
    public function __construct()
    {
    
    }

    /**
     * @return crossmapSetMembers
     */
    public function getCrossmapSetMembers()
    {
      return $this->crossmapSetMembers;
    }

    /**
     * @param crossmapSetMembers $crossmapSetMembers
     * @return CrossmapSetMembersResponse
     */
    public function setCrossmapSetMembers($crossmapSetMembers)
    {
      $this->crossmapSetMembers = $crossmapSetMembers;
      return $this;
    }

}
